db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -411,6 +414,7 @@ Status DBImpl::AtomicFlushMemTablesToOutputFiles(
   }
 
   std::vector<FileMetaData> file_meta(num_cfs);
+  std::vector<uint8_t> switched_to_mempurge(num_cfs, false);
```

Comment:
Shouldn't it be 0 instead of false? I don't think it will make any difference apart from readability.

Reply:
My bad, thanks for catching that, I was trying to use `bool`s the whole time so that's just a remnant of what I used before switching to `uint8_t`.